src/azure-cli-core/azure/cli/core/authentication.py

```diff
@@ -25,7 +25,7 @@ def _create_scopes(resource):
         scope = resource + '/.default'
     else:
         scope = resource.rstrip('/') + '/.default'
```

Comment:
@qianwens, let's not hard-code `datalake`, `batch` and `database`, but `rstrip` anyway.